### ISSUE TITLE
Internal storage folder structure is messed up in recovery

### ISSUE THREAD

**Issue by m3thm4th** [SharkBoy69-CS/Lineage_OTA#7]
Folder structure is messed up in recovery, only for internal storage (`/sdcard` folder).

I installed this ROM, went through the initial wizard, didn't set up any screen lock (no PIN or else), rebooted to OrangeFox recovery and found folder structure is completely messed up, folders names are alphanumeric strings, there are sub-folders and files, also named with alphanumeric strings (there should actually be only the empty default folders).

This is the case with another LOS21 ROM with San Kernel R1.1 (4.19) I also tried ([link here](https://4pda.to/forum/index.php?showtopic=946312&st=71760#entry131749580)), this issue doesn't happen with another LOS21 ROM with kernel 4.4 i found [here](https://www.pling.com/p/1889410/) (the only working LOS ROM from these is the `gsh4b2qb2wv7`, the link is dead, but it can be found [here](https://sourceforge.net/projects/spidybuilds/files/lavenderROMs/LineageOSExt/lineage-21.0-Ext-Community-lavender-Vanilla-20240503-1718.zip/download), all the other LOS versions result in bootloop after clean flash).

I don't think the kernel version should give such issues, but that's the only difference I'm aware of. Also, the OrangeFox recovery version I'm using is the R11.1_6_1_kernel_419 which supports kernel 4.19 ([link](https://orangefox.download/device/lavender)).

**Comment by 0xsharkboy** [comment]
Congrats, you discovered rom encryption

**Comment by m3thm4th** [comment]
But encryption uses the screen lock as the key, since there is no screen lock set, there should be no encryption. And even then, OrangeFox can handle encryption, but it doesn't even ask for a key. Beside this, it is only `/sdcard` folder that is encrypted, all other folders (`/system`, `/data`, etc.) are all accessible, so it makes no sense anyway.

**Comment by 0xsharkboy** [comment]
Decryption is [not supported](https://orangefox.download/release/6687022438916e89ef63db4d/bugs)

**Comment by m3thm4th** [comment]
On previous builds by SpiDy I was able to browse internal storage normally with the same OrangeFox, so somehow it has to do with the ROM itself. I tried wiping and formatting data to no avail, Internal storage stays encrypted no matter what, no amount of wiping can fix this, aalthough many users reported getting rid of encryption after formatting data. This is a very important feature and it's needed for flashing zips without using a PC or an sd card. Is there a way to access internal storage from recovery that you know of?
